### ISSUE TITLE
All labels in text_WithLabel.txt of ColonPath are 0?

### ISSUE THREAD

**Issue by zihou98** [openmedlab/MedFM#1]
It seems that labels of all sample recorded in the test_WithLabel.txt of the ColonPath dataset are 0. Is this correct?

**Comment by MengzhangLI** [comment]
Hi, thanks for your issue.

It is correct but we add 1,000 images whose labels are `1` from validation part to training part.

We move 1,000 images in Colon validation set whose labels are 1 to training set:
https://medfm2023.grand-challenge.org/datasets/

Please check it out:

[test_WithoutLabel.txt](https://github.com/openmedlab/MedFMC/files/11634512/test_WithoutLabel.txt)

[test_WithLabel.txt](https://github.com/openmedlab/MedFMC/files/11634510/test_WithLabel.txt)

We would renew github repo asap.

Best,
